templates/index.html

```diff
@@ -8,11 +8,11 @@
 {% block meta_copydoc %}https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit{% endblock %}
 
 {% block content %}
-<script defer src="{{ versioned_static('js/copy-to-clipboard.js') }}"></script>
+
 <script defer src="{{ versioned_static('js/osselector.js') }}"></script>
 
 <main id="main-content">
-  <section class="p-strip--suru is-dark">
+  <section class="p-strip--suru is-dark is-deep">
```

Comment:
Not sure why this was added, without it the strip seems to work OK at each breakpoint?